docs/AngularJS/angular-hybrid.md

```diff
@@ -9,7 +9,7 @@ position: 5
 
 # AngularJS for Hybrid UI
 
-As of the Kendo UI Q3 2014 release, Kendo UI includes AngularJS directives for some of its Hybrid UI components, such as the hybrid mobile Application and View, the SplitView, and the ModalView. In this context, the mobile views support AngularJS controllers, directives and tw- way data binding expressions.
+As of the Kendo UI Q3 2014 release, Kendo UI includes AngularJS directives for some of its Hybrid UI components, such as the hybrid mobile Application and View, the SplitView, and the ModalView. In this context, the mobile views support AngularJS controllers, directives and the- way data binding expressions.
 
```

Comment:
Yes it is detected with the tool, and will be fixed in master branch, this branch is solely for setting .spelling file which holds false positives.